api-server/src/announcements/announcements.controller.ts

```diff
@@ -31,7 +31,7 @@ export class AnnouncementsController {
   }
 
   @Post('/ensure-one-announcement')
-  @UseGuards(AuthGuard)
+  @UseGuards(AdminAuthGuard)
```

Comment:
We no have to be more explicit in this controller, all of the endpoints require that you are at least authenticated, but some require full infinite-admin access while others require only that you a partner or an infinite-admin

I added quite a bit more testing around this and other auth scenarios as it was all getting quite confusing, and found some preexisting bugs in the process